graphbolt/src/cuda/expand_indptr.cu

```diff
@@ -54,8 +62,8 @@ struct AdjacentDifference {
 
 torch::Tensor ExpandIndptrImpl(
     torch::Tensor indptr, torch::ScalarType dtype,
-    torch::optional<torch::Tensor> nodes,
-    torch::optional<int64_t> output_size) {
+    torch::optional<torch::Tensor> nodes, torch::optional<int64_t> output_size,
+    const bool edge_ids) {
```

Comment:
include edge ids? has edge ids?

Reply:
Alright, let me change to a better name.